debian/copyright

```diff
@@ -273,7 +339,9 @@ License: W3C
 Files: debian/*
 Copyright: 2010 Ivan Wong <ivanwyc@gmail.com>
            2010-2013 Ondřej Surý <ondrej@debian.org>
-           2013 Michael Stapelberg <stapelberg@debian.org>
+           2012-2014 Michael Stapelberg <stapelberg@debian.org>
+           2014-2015 Canonical Ltd
+           2014-2015 Tianon Gravi <admwiggin@gmail.com>
```

Comment:
See https://github.com/i3/i3/pull/1636 — in i3, we’ve gotten rid of copyright ranges entirely, see the PR for rationale. Maybe worth doing here as well?